api/jsonapi_test.go

```diff
@@ -440,7 +444,7 @@ func TestCryptoListingsIllegalFields(t *testing.T) {
 	physicalListing := factory.NewListing("physical")
 
 	listing := factory.NewCryptoListing("crypto")
-	listing.Metadata.PricingCurrency = "btc"
+	listing.Metadata.PricingCurrency = &pb.CurrencyDefinition{Code: "BTC", Divisibility: 8}
```

Comment:
The preferred way to create CurrencyDefinitions is with `repo.LoadCurrencyDefinitions().Lookup("BTC")`.